symfony/plugins/orangehrmTimePlugin/test/fixtures/testcases/MyTimesheetItemAPITestCases.yaml

```diff
@@ -284,7 +284,21 @@ GetAll:
         "middleName": "",
         "employeeId": "0024",
         "terminationId": null
-      }
+      },
```

Comment:
add more test cases to cover user role allowed actions.

Reply:
covered for supervisor role, admin and ESS are already covered